src/kernels/jupyter/types.ts

```diff
@@ -243,16 +253,10 @@ export interface IJupyterUriProviderRegistration {
 export interface IJupyterServerUriEntry {
     /**
      * Uri of the server to connect to
+     * @deprecated
      */
-    uri: string;
-    provider: {
-        id: string;
-        handle: JupyterServerUriHandle;
-    };
-    /**
-     * Unique ID using a hash of the full uri
-     */
-    serverId: string;
+    uri?: string;
```

Comment:
This property is now deprecated, it had a number of meanings in different contexts
Hence very buggy